lib/Driver.cpp

```diff
@@ -63,25 +63,34 @@ std::vector<std::unique_ptr<TestResult>> Driver::Run() {
 
   for (auto &Test : Finder.findTests(Ctx)) {
     auto ObjectFiles = AllObjectFiles();
-    ExecutionResult ExecResult = Runner.runTest(Test.get(), ObjectFiles);
+
+    ExecutionResult ExecResult = Sandbox.run([&](ExecutionResult *SharedResult){
+      *SharedResult = Runner.runTest(Test.get(), ObjectFiles);
+    });
+
     auto BorrowedTest = Test.get();
     auto Result = make_unique<TestResult>(ExecResult, std::move(Test));
 
     for (auto Testee : Finder.findTestees(BorrowedTest, Ctx)) {
       auto ObjectFiles = AllButOne(Testee->getParent());
       for (auto &MutationPoint : Finder.findMutationPoints(MutationOperators, *Testee)) {
+        ExecutionResult R = Sandbox.run([&](ExecutionResult *SharedResult){
```

Comment:
I do not know if the test framework is "pre booted" in mutang, I just want to mention at this point that fork based sandboxing (I call it isolation in mutant) comes at a non obvious cost: If killing the mutation has a one time setup cost, like scanning test files, or anything that would normally happen *once* for N kills, now gets payed *per* kill. As this one time setup work has to be re-done per sandbox.

For this reason in mutant I try to trigger all "one time" work before attempting mutation kills, to inherit that one time work with the fork.

If this strategy is even required in mutangs case I should note here that there may need to be a hook so test frameworks can re-connect the database to unshare FDs (assuming the test framework on performing the one-time work would create such).